src/static/helpers/types.ts

```diff
@@ -43,3 +43,14 @@ export interface QueryParameters {
  * Generic interface for all parameter types.
  */
 export type UrlParameters = PathParameters | QueryParameters;
+
+/* eslint-disable no-shadow */
+// temporary fix for eslint bug: https://github.com/typescript-eslint/tslint-to-eslint-config/issues/856
+/**
+ * Enum type representing different return options
+ */
+export enum returnType {
+  JSON,
+  RESPONSE,
+  REQUEST,
+}
```

Comment:
```suggestion
export type MethodReturnType = "parsed" | "request" | "response";
```
1. So I thought a bit more about this, and I think that a union type is cleaner than an enum here, because that way users can provide just a string, rather than needing to import/use the enum object. [Here's an article](https://blog.bam.tech/developer-news/should-you-use-enums-or-union-types-in-typescript) that weighs pros/cons of enum vs union type. 
2. Convention for enums is to use UpperCamelCase. But `ReturnType` is a built in type, so use a different name. `MethodReturnType` is the first thing that came to mind, but I'm not sure it's the best option. Naming things is hard.
3. `json` -> `parsed` just in case we ever get something where the response body is _not_ JSON, but the user wants us to handle it, rather than receive the raw response.